Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -4300,17 +4300,17 @@ MaskWebGLStringsEnabled:
 
 MasonryEnabled:
   type: bool
-  status: preview
+  status: stable
   category: css
   humanReadableName: "CSS Masonry Layout"
   humanReadableDescription: "Enable Masonry Layout for CSS Grid"
   defaultValue:
     WebKitLegacy:
       default: false
```

Comment:
Probably doesn't hurt to enable in WebKitLegacy too given this is mostly WebCore. No strong opinion though

Reply:
That is fine; turned it on in WebKitLegacy.